api/api_sources/schema-migration-sql/CountryProvinceSchema/CountryProvinceSchema.sql

```diff
@@ -0,0 +1,37 @@
+-- ### Creating Table: country_province ### --
+
+        
+CREATE TABLE country_province ();
+ALTER TABLE country_province ADD COLUMN province_code VARCHAR(2) NOT NULL;
+ALTER TABLE country_province ADD COLUMN country_code VARCHAR(3) NOT NULL;
+ALTER TABLE country_province ADD COLUMN description VARCHAR(100) NULL;
+
+
+        
+-- ### Creating Comments on table ### --
+
+        
+COMMENT ON TABLE country_province IS 'Standard ISO-3166 code table for listed countries and provinces';
+COMMENT ON COLUMN country_province.province_code IS 'ISO-3166 standard two character identifier for province name. With country code generate primary key of the table';
```

Comment:
Minor update to column comment:
"ISO-3166 standard two character sub-division identifier for provinces, states, and territories. Combines with country code to generate primary key of the table"